config/main.py

```diff
@@ -1247,6 +1265,25 @@ def load_minigraph(db, no_service_restart):
         else:
             command = "{} -H -m --write-to-db {}".format(SONIC_CFGGEN_PATH, cfggen_namespace_option)
         clicommon.run_command(command, display_cmd=True)
+
+        if namespace is DEFAULT_NAMESPACE:
+            npu_id = ''
+        else:
+            npu_id = device_info.get_npu_id_from_name(namespace)
+
+        # load default_config.json file from platform directory
+        default_config_file = os.path.join('/usr/share/sonic/device/', platform, npu_id, 'default_config.json')
```

Comment:
can you give an example what  is present in 'default_config.json'

Reply:
I added the following example in the description:
```json
{
    "DEVICE_METADATA": {
        "localhost": {
            "instance_name": "ASIC0",
            "chassis_db_address" : "240.127.1.1",
            "start_chassis_db" : "1",
            "asic_id": "06:00.0"
        }
    }
}
```

Reply:
> As per PR [Azure/sonic-buildimage#5991](https://github.com/Azure/sonic-buildimage/pull/5991), for VOQ chassis systems the "asic_id" and "instance_name" are configured via minigraph.xml. Will these values from default_config.json be preferred over what is configured through minigraph?

Both are still valid. If the platform has configuration in `default_config.json` it has to be static. If it is present in both files then `default_config.json` will override the value configured via `minigraph.xml`, which is fine.

The example above is probably incorrect once VoQ config using minigraph.xml is complete ([PR5991](https://github.com/Azure/sonic-buildimage/pull/5991)), however `asic_id` is still required in this file as it is platform-dependent.

Reply:
> For VOQ chassis systems, we get "start_chassis_db" and "chassis_db_address" from chassisdb.conf from device specific directory (for example: https://github.com/Azure/sonic-buildimage/blob/master/device/arista/x86_64-arista_7800_sup/chassisdb.conf). These are needed before the database service is started in supervisor card. Having these in config_db is of no use. Also, if these are not used in multi-asic we should not have these in default_config.json.

Right. Actually in the above example only `asic_id` is required as it is tied to each asic physically on the hardware. Other configurations can easily be set using minigraph.xml